synapse/config/appservice.py

```diff
@@ -47,6 +48,12 @@ def generate_config_section(cls, **kwargs: Any) -> str:
         # enables MAU tracking for application service users.
         #
         #track_appservice_user_ips: true
+
+        # Set to establish maximum backoff (in seconds) between HS -> AS connection attempts.
+        # Upon failing to push appservice events the homeserver will wait
+        # an increasing amount of seconds between retries. This sets an upper limit of that (in seconds)
+        #
+        #appservice_max_backoff: 60
```

Comment:
> Broadly looks good, thanks. Could you add an entry to the [config manual](https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html) for this option, please? The source is https://github.com/matrix-org/synapse/blob/develop/docs/usage/configuration/config_documentation.md

This seems to be duplicating information from the sample config – for appservice configs it seems to be copied from it almost verbatim. If you didn't tell me otherwise, I'd assume it was autogenerated this way :)

Is there any plan/effort to make it generated, or does it serve some broader purpose? Is it expected/acceptable to just copypaste the description from config sample?

Reply:
Done now in 727bbf855